docs/developer/developing.md

```diff
@@ -0,0 +1,46 @@
+# Developing Cincinnati
+
+## Overview
+
+The following sections describe how to build and test the project for local development. 
+
+
+## Prerequisites
+
+* Git
+* Rust
+* [just](https://github.com/casey/just)
+
+*Note:* We recommend using [Rustup](https://github.com/rust-lang/rustup/blob/master/README.md) for Rust toolchain management. 
+
+## Running it locally
+
+Please below commands in parallel (for e.g. in different terminals at the same time).
+
+```shell
+just run-graph-builder
+```
+
+```shell
+just run-policy-engine
+```
+
+Here is the command to get the graph for stable-4.2 amd64 graph architecture
```

Comment:
Sorry if [my earlier recommendation][1] was unclear, but I think you can drop "architecture" here.  If you want to keep it, it should move closer to amd64 and not have "graph" in between.

[1]: https://github.com/openshift/cincinnati/pull/198#discussion_r370886260

Reply:
Fixed in 1ed279c.